admin-dev/themes/new-theme/js/app/pages/stock/components/overview/product-line.vue

```diff
@@ -67,7 +73,7 @@
       </span>
     </td>
     <td class="qty-spinner">
-      <Spinner :product="product" class="float-xs-right" />
+      <Spinner :product="product" class="pull-xs-right" @updateProductQty="updateProductQty" />
```

Comment:
`pull...` does not work with bootstrap 4, please upgrade your node modules. Also `-xs-` becomes `sm`